docs/self-managed/update/administrators/plan-update.md

```diff
@@ -0,0 +1,237 @@
+---
+id: plan-admin-update
+title: "Plan an update"
+description: "Plan your update to Camunda 8.8 on Self-Managed - Administrator guide."
+---
+
+A successful Camunda 8.8 update requires thorough planning. This guide helps you assess your infrastructure, understand operational requirements, and choose the right update strategy for your environment.
+
+## Step 1: Identify your upgrade path
+
+Determine your current and target versions to plan the update path.
+
+### Current version assessment
+
+**First, identify your current Camunda versions:**
+
+```bash
+# Check Kubernetes deployment versions
+kubectl get pods -l app.kubernetes.io/instance=camunda-platform -o jsonpath='{range .items[*]}{.metadata.name}{"\t"}{.spec.containers[0].image}{"\n"}{end}'
+
+# Check Helm chart version
+helm list | grep camunda-platform
+```
+
+### Sequential update requirement
```

Comment:
Maybe link to https://docs.camunda.io/docs/next/self-managed/operational-guides/update-guide/introduction/#update-procedure, which is our central place to document this procedure?

Reply:
@theburi removed the file, resolving